lib/at_cmd_parser/at_cmd_parser.c

```diff
@@ -343,3 +340,29 @@ int at_parser_max_params_from_str(const char *at_params_str,
 
 	return err;
 }
+
+enum at_cmd_type at_parser_cmd_type_get(const char *at_cmd)
+{
+	enum at_cmd_type type;
+
+	if (!is_command(at_cmd)) {
+		return AT_CMD_TYPE_UNKNOWN;
+	}
+
+	skip_command_prefix(&at_cmd);
+
+	while (is_valid_notification_char(*at_cmd)) {
+		at_cmd++;
+	}
+
```

Comment:
I think there is a need to check CRLF or Terminator here. If yes, then SET type command without parameter.